_docker/lib/backup/backup.rb

```diff
@@ -0,0 +1,112 @@
+require 'date'
+
+require_relative '../../../_docker/lib/default_logger'
+require_relative '../../../_docker/lib/process_runner'
+require_relative 'git_backup_strategy'
+
+#
+# The purpose of this class is to perform a backup of a RHD environment.
+# It can be passed a single argument, which will name the folder into which the backup is placed.
+#
+# @author rblake@redhat.com
+#
+class Backup
+
+  # Default name of the Drupal filesystem backup
+  @@DRUPAL_FILESYSTEM_TAR = 'drupal-filesystem.tar.gz'
```

Comment:
Is this class used multiple times? I don't see a reason to use class variables for this versus instance variables